dev/build.image/publish/wlp/templates/servers/springBoot3/server.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<server description="new server">
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>servlet-6.0</feature>
+        <feature>pages-3.1</feature>
+        <feature>webSocket-2.1</feature>
+
+        <feature>transportSecurity-1.0</feature>
+
+        <feature>springBoot-3.0</feature>
```

Comment:
Re: The missing servlet and websockets tests not working doesn't make sense to me. It would seem that the started server has features in it still for some reason.
I was able to get those working.  I had problems in my update to FeatureAuditor.

Reply:
Re: Messaging changes.  I have the same worry.  The difficulty here is that the existing messages don't work very well when adding in the spring3 tests.  Also, the existing messages are not quite accurate.

Reply:
ok, the sprint2 broken missing servlet and missing websocket tests are working.  What remains broken are the spring3 missing servlet and missing websocket tests.  What am seeing is that jakarta Servlet and jakarta WebSocket are loadable with just the springBoot-3.0 feature set.  That means the condition for emitting the missing feature message is never triggerred, and the tests fail.